src/platform/forms-system/src/js/containers/FormApp.jsx

```diff
@@ -24,25 +30,30 @@ class FormApp extends React.Component {
   render() {
     const { currentLocation, formConfig, children, formData } = this.props;
     const trimmedPathname = currentLocation.pathname.replace(/\/$/, '');
+    const lastPathComponent = currentLocation.pathname.split('/').pop();
     const isIntroductionPage = trimmedPathname.endsWith('introduction');
+    const isNonFormPage = this.nonFormPages.includes(lastPathComponent);
     const Footer = formConfig.footerContent;
 
     let formTitle;
     let formNav;
     let renderedChildren = children;
-    if (!isIntroductionPage) {
-      // Show title only if we're not on the intro page and if there is a title
-      // specified in the form config
+    if (!isIntroductionPage && !isNonFormPage) {
+      // Show title only if:
+      // 1. we're not on the intro page *or* one of the additionalRoutes
+      //    specified in the form config
+      // 2. there is a title specified in the form config
       if (formConfig.title) {
         formTitle = (
           <FormTitle title={formConfig.title} subTitle={formConfig.subTitle} />
         );
       }
     }
 
-    // Show nav only if we're not on the intro, form-saved, error, or confirmation page
+    // Show nav only if we're not on the intro, form-saved, error, confirmation
+    // page or one of the additionalRoutes specified in the form config
     // Also add form classes only if on an actual form page
-    if (isInProgress(trimmedPathname)) {
+    if (!isNonFormPage && isInProgress(trimmedPathname)) {
```

Comment:
These were the changes to `us-forms-system` you had originally made to support additional routes. Now that the code has moved back to `vets-website`, I went ahead and applied those changes again for this PR.

Reply:
Oh, haha, ignore the comment I just made! Thanks for doing this!